iree/turbine/kernel/wave/utils.py

```diff
@@ -641,7 +642,8 @@ def bfs(
     filter_fn: Callable[[fx.node], bool],
 ) -> set[fx.Node]:
     """
-    Run BFS on the graph to capture the forward slice of a node.
+    Run BFS on the graph. The filter function is not applied to
+    the incoming node.
```

Comment:
Is this still needed?

Reply:
Just updating the comment to be more accurate and specify that the filter function is not applied to the incoming note.

Reply:
ah right, that makes sense!